content/doc/book/scaling/scaling-jenkins-on-kubernetes.adoc

```diff
@@ -1,15 +1,235 @@
 ---
 layout: section
-wip: true
 ---
 ifdef::backend-html5[]
 :doctitle: Scaling Jenkins on Kubernetes
 :notitle:
 :description:
 :author:
 :email: jenkinsci-users@googlegroups.com
+:imagesdir: /doc/book/resources/scaling-jenkins-on-kubernetes
 :sectanchors:
 :toc: left
 endif::[]
 
-= Scaling Jenkins on Kubernetes
\ No newline at end of file
+= Scaling Jenkins on Kubernetes
```

Comment:
I went through the guide, and TBH I do not see scaling guidelines. There are some relevant bits like adding agents, but the rest rather describes deploying Jenkins on K8s. Would it make sense to rename the page or restructure the content?